### DISCUSSION TITLE
Lost progress of "todo's" when trying to rebuild container

### DISCUSSION THREAD

**Discussion by Frostydosty** [tudelft/AE4353-Y24#14] [General]
Hello,

When trying to commit my changes when finishing exercise 1, it prompted me to rebuild the container. However, when it got stuck, I cancelled the rebuilding. Now exercise 1 is in the template version again, but my devcontainer and other changes are still visible/there. Is there a way to save the edits I made for ex.1?

Thank you in advance.

**Comment by yilun-wu** [comment]
Are you on Windows? If so, can you see the edits you made in files from ex_1 in WSL?

**Reply by Frostydosty** [reply]
You mean in the VSC timeline? This is the only thing I can see
![Schermafbeelding 2024-09-12 154742](https://github.com/user-attachments/assets/18090895-ec27-4c92-a5be-a536cdf894e0)
